src/EditorFeatures/Core/Implementation/IntelliSense/QuickInfo/Controller.cs

```diff
@@ -14,20 +13,25 @@
 using Microsoft.VisualStudio.Language.Intellisense;
 using Microsoft.VisualStudio.Text;
 using Microsoft.VisualStudio.Text.Editor;
+using Microsoft.VisualStudio.Text.Editor.Commanding;
+using Microsoft.VisualStudio.Text.Editor.Commanding.Commands;
 using Roslyn.Utilities;
+using VSCommanding = Microsoft.VisualStudio.Commanding;
 
 #pragma warning disable CS0618 // IQuickInfo* is obsolete
 namespace Microsoft.CodeAnalysis.Editor.Implementation.IntelliSense.QuickInfo
 {
     internal partial class Controller :
         AbstractController<Session<Controller, Model, IQuickInfoPresenterSession>, Model, IQuickInfoPresenterSession, IQuickInfoSession>,
-        ICommandHandler<InvokeQuickInfoCommandArgs>
```

Comment:
> ICommandHandler<InvokeQuickInfoCommandArgs> [](start = 7, length = 44)

actually dead code, QuickInfo is invoked by the editor since 15.3 and Roslyn  has no command handler for this command